### ISSUE TITLE
IndexWhere Returns Incorrect Value When Starting Index is Negative on Scala 2.11.12

### ISSUE THREAD

**Issue by adamgfraser** [scala/bug#11852]
```scala
val as = List("")
as.indexWhere(_ => true, -10) // should be 0 but -10 on Scala 2.11.12
```
https://scastie.scala-lang.org/KSokOeYES3Wg6UE44l9psQ

It looks like there have been several similar issues in the past but none of them are still open and as Scala 2.11.12 is still being supported at this point, as I understand it, raising again.

**Comment by SethTisue** [comment]
this was fixed in Scala 2.12.2, so, closing. perhaps the bug report will be valuable to someone as documentation. (it is unlikely there will be any further 2.11.x releases.)

**Comment by adamgfraser** [comment]
Okay. Thanks for the feedback.
